src/ansys/geometry/core/designer/designpoint.py

```diff
@@ -89,4 +90,8 @@ def __repr__(self) -> str:
         return "\n".join(lines)
 
     def _to_polydata(self) -> pv.PolyData:
-        return pv.Sphere(center=self.value.flat, radius=0.001)
+        """Get polydata from DesignPoint object."""
+        # get units to plot proportionally
+        ureg = UnitRegistry()
+        unit = 1 * self.value.unit
```

Comment:
Nope, use our UNITS object. Do not redeclare a ``UnitRegistry``. This will break things